src/vsock/src/rxops.rs

```diff
@@ -0,0 +1,22 @@
+#![deny(missing_docs)]
+
+#[derive(Clone, Copy, PartialEq, Debug)]
```

Comment:
Do you really need all of these? I am not sure if rustc complains about unused traits. I would remove whatever traits the code isn't using. This applies for all structures.